src/barril/units/posc.py

```diff
@@ -326,6 +327,8 @@ def FillUnitDatabaseWithPosc(db=None, fill_categories=True, override_categories=
     db.AddUnitBase("force per velocity", "Newton second per meter", "Ns/m")
     db.AddUnitBase("force per angle", "Newton per angle", "N/rad")
     db.AddUnitBase("force per angular velocity", "Newton second per angle", "Ns/rad")
+    db.AddUnitBase('moment per angle', 'Newton meter per angle', 'Nm/rad')
```

Comment:
Please follow the instructions on the README and install `pre-commit` to run black-formatting on the code automatically. 👍